integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/IntegrasjonspunktApplication.java

```diff
@@ -1,7 +1,10 @@
 package no.difi.meldingsutveksling;
 
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
 import com.sun.xml.ws.transport.http.servlet.WSSpringServlet;
+import no.difi.meldingsutveksling.config.IntegrasjonspunktProperties;
 import no.difi.meldingsutveksling.config.IntegrasjonspunktPropertiesValidator;
+import no.difi.meldingsutveksling.nextmove.NextMoveServiceBus;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'no.difi.meldingsutveksling.nextmove.NextMoveServiceBus'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AUselessImportCheck)